### PR TITLE
Hide processing thousand validator program under feature flag

### PR DESCRIPTION
**Pull request by MRamanenkau**
https://cerenetwork.atlassian.net/browse/CBI-2105

### CODE REVIEW

File: backend/crawlers/ranking.js [hunk 1]
```diff
@@ -9,10 +9,13 @@ const {
   wait,
   dbQuery,
   dbParamQuery,
+  getCrawlerConfigByName,
 } = require('../lib/utils');
 const backendConfig = require('../backend.config');
 
 const crawlerName = 'ranking';
+const crawlerConfig = getCrawlerConfigByName(backendConfig, crawlerName);
```
Comment (AndreiNavoichyk) [inline backend/crawlers/ranking.js]: `config` is already in place. Please, use it

File: backend/backend.config.js [hunk 2]
```diff
@@ -46,6 +46,7 @@ module.exports = {
       erasPerDay: 1,
       tokenDecimals: 10,
       featuredTimespan: 60 * 60 * 24 * 7 * 2 * 1000, // 2 weeks
+      isThousandValidatorsEnabled: process.env.IS_THOUSAND_VALIDATORS_ENABLED || false,
```
Comment (AndreiNavoichyk) [inline backend/backend.config.js]: Let's rename to `isThousandValidatorsProgrammeEnabled`

File: backend/crawlers/ranking.js [hunk 3]
```diff
@@ -604,9 +607,13 @@ const crawler = async (delayedStart) => {
     logger.debug(loggerOptions, `Last era in DB is ${lastEraInDb}`);
 
     // thousand validators program data
-    logger.debug(loggerOptions, 'Fetching thousand validator program validators ...');
-    const thousandValidators = await getThousandValidators();
-    logger.debug(loggerOptions, `Got info of ${thousandValidators.length} validators from Thousand Validators program API`);
+    // https://wiki.polkadot.network/docs/thousand-validators
+    let thousandValidators;
```
Comment (AndreiNavoichyk) [inline backend/crawlers/ranking.js]: Let's use an empty array here. So you don't need to add extra checks in the code below

**Review by AndreiNavoichyk — COMMENTED** [review]
Please, update CHANGELOG

File: backend/crawlers/ranking.js (hunk 1 above)
Comment (MRamanenkau) [reply backend/crawlers/ranking.js]: Done

File: backend/backend.config.js (hunk 2 above)
Comment (MRamanenkau) [reply backend/backend.config.js]: Renamed

File: backend/crawlers/ranking.js (hunk 3 above)
Comment (MRamanenkau) [reply backend/crawlers/ranking.js]: Done

**Comment by MRamanenkau** [comment]
> Please, update CHANGELOG

Done

File: backend/backend.config.js (hunk 2 above)
Comment (AndreiNavoichyk) [reply backend/backend.config.js]: Can not see changes.

Comment (MRamanenkau) [reply backend/backend.config.js]: Updated

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**